<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>529</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Hack Nerd Font</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ground Station</string>
  </property>
  <widget class="PlotWidget" name="Graph_Altitud" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="PlotWidget" name="Graph_TemperaturaInterna" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="PlotWidget" name="Graph_Velocidad" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="PlotWidget" name="Graph_Temperatura_Externa" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>321</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="PlotWidget" name="Graph_Humedad" native="true">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tiempo :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>15 min</string>
   </property>
  </widget>
  <widget class="QLabel" name="Battery_Label">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>490</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string></string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Altitud</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Velocidad</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>210</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Humedad</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Giro</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Temperatura Interna</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboPorts">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font Mono</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Puertos Seriales</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Record_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FiraCode Nerd Font Mono</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonPorts">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string></string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
